mlflow/server/js/.eslintrc.js

```diff
@@ -146,7 +148,7 @@ module.exports = {
     'no-catch-shadow': "off",
     'no-class-assign': 2,
     'no-cond-assign': 2,
-    'no-confusing-arrow': [2, { 'allowParens': true }],
+    'no-confusing-arrow': 0,
```

Comment:
@Zangr I disabled some ESlint rules that conflict with prettier. Are you fine with these changes?

Reply:
Could you please comment on each of the disable eslintrc rules with the conflicting prettier rule?

Reply:
Got it. Prettier removes () here, which conflicts with [no-confusing-arrow](https://eslint.org/docs/rules/no-confusing-arrow).

<img width="557" alt="Screen Shot 2020-04-26 at 17 37 10" src="https://user-images.githubusercontent.com/17039389/80302362-96c4e200-87e4-11ea-99e1-0b01f8491a64.png">

Reply:
This lint rule makes sense. Let's keep it by removing the conflicting prettier rule.

Reply:
Prettier removes unnecessary parentheses and it's not possible to keep `()` here. There are three options to get around with this.

1. Don't use Prettier.
2. Insert `// prettier-ignore` as a comment to make Prettier ignore these lines.
3. Disable ESLint's no-confusing-arrow

Reply:
I see. I think overall the benefit prettier provides overrules adjustments of that legacy lint rules. As far as we run `npm run lint:fix` for each commit, it should be fine.